src/net/sourceforge/kolmafia/persistence/AdventureQueueDatabase.java

```diff
@@ -236,24 +238,14 @@ public static void deserialize() {
       AdventureQueueDatabase.checkZones();
     } catch (FileNotFoundException e) {
       AdventureQueueDatabase.resetQueue(false);
-      return;
-    } catch (ClassNotFoundException e) {
+    } catch (ClassNotFoundException | EOFException | ClassCastException e) {
       // Found the file, but the contents did not contain a properly-serialized treemap.
       // Wipe the bogus file.
       file.delete();
       AdventureQueueDatabase.resetQueue();
-      return;
-    } catch (ClassCastException e) {
-      // Old version of the combat queue handling.  Sorry, have to delete your queue.
-      file.delete();
-      AdventureQueueDatabase.resetQueue();
-      return;
-    } catch (EOFException e) {
-      // Malformed data. Wipe the bogus file.
-      file.delete();
-      AdventureQueueDatabase.resetQueue();
-      return;
-    } catch (IOException e) {
+    } // Old version of the combat queue handling.  Sorry, have to delete your queue.
```

Comment:
this combination has put the comments in the wrong place.

Reply:
ok